### ISSUE TITLE
Tarr fails health checks on some files (missing transcode args)

### ISSUE THREAD

**Issue by DasPflanze** [HaveAGitGat/Tdarr#673]
I'm currently using tdarr to convert TV recordings made by Plex from h264 to h265 and also to check the health of the files to see if some files have issues because of bad satelite reception. My setup worked for like 2 weeks without issues. Now for all new recorded files tdarr fails the health checks and they get listed under "Health check: Error/Cancelled". Old files can still be checked.
All files that won't work with the health checks are already transcoded to h265 by tdarr, so there shouldn't be an permission issue.

**Expected behavior**
It should run the health check and list the files in "Health check: Healthy" or "Health check: Error/Cancelled" accordingly.

**Please provide the following information:**
- Config files are attached as .txt, as I couldn't upload .json
[Tdarr_Server_Config.txt](https://github.com/HaveAGitGat/Tdarr/files/9392149/Tdarr_Server_Config.txt)
[Tdarr_Node_Config.txt](https://github.com/HaveAGitGat/Tdarr/files/9392148/Tdarr_Node_Config.txt)

- Log files 
[Tdarr_Server_Log.txt](https://github.com/HaveAGitGat/Tdarr/files/9392141/Tdarr_Server_Log.txt)
[Tdarr_Node_Log.txt](https://github.com/HaveAGitGat/Tdarr/files/9392145/Tdarr_Node_Log.txt)

- Worker error (line 28 and 37 show no args)
[BrnxSUzH2B1-log.txt](https://github.com/HaveAGitGat/Tdarr/files/9392154/BrnxSUzH2B1-log.txt)

 - OS: Docker on unraid
 - Browser chrome
 - Version 2.00.18

**Comment by DasPflanze** [comment]
Here is the scan of a working file:
[SantaBaby2.txt](https://github.com/HaveAGitGat/Tdarr/files/9392212/SantaBaby2.txt)
And one of a not working file:
[MarvelsTheAvengers.txt](https://github.com/HaveAGitGat/Tdarr/files/9392217/MarvelsTheAvengers.txt)

**Comment by HaveAGitGat** [comment]
Can try downloading some h265/h265 samples from here:

https://tdarr.io/samples/

E.g. this one:

https://samples.tdarr.io/api/v1/samples/sample__1080__libx265__aac__30s__video.mkv

Try running the health check on that. Will help see if the issue is with your files or something else.

**Comment by DasPflanze** [comment]
So I guess I found a solution. I had multiple librarys that only did transcoding (health checks off):

- Transcode only .ts to h265 in /movies

- Transcode only .ts to h265 in /TV movies

- Transcode only .ts to h265 in /series

As I only wanted to transcode recordings that were made by Plex. And I had 3 more librarys that only did health checks:

- on /movies

- on /TV movies

- on /series

Because all libraries that transcoded only scanned .ts files the health check would also only be run on .ts files. At the beginning everything went fine but now the "health check"-only libraries wouldn't process anything anymore and somehow one of the "transcode"-only libraries (TV movies) added the transcoded files to the health check list and did not generate the command for health checking. Maybe the other libraries would add their files also to the health check queue but I did not add new files to them.

So for now I disabled the "health ckeck"-only libraries and enabled health checks on the other libraries so at least the newly added .ts files get transcoded to .mkv and then health checked.

It's a bit wierd...but for now it works. Sadly I can't health check my existing files.

Possible feature request:
It would be very nice if we could add health checks (like a plugin) to the transcoding process so if a file would have errors it wouldn't even be processed in the first place. Right now I'm transcoding my recordings from .ts to .mkv, so that they get then health checked and maybe then get deleted because there are errors. So the transcoding was completely wasted.
